### ISSUE TITLE
Search started when user is noticed with warning message (reduce area or keywords amount)

### ISSUE THREAD

**Issue by Stref** [CUAHSI/HISWebClientIssues#16]
_Steps to reproduce:_
1. Navigate to http://qa-hiswebclient.azurewebsites.net/
2. Select 2 any keywords
3. Select all services
4. Select area 25 sq  km+
5. Click Search Map

_Result:_
Warning message appeared but search doesn't stoped  (see screenshot 747) 

_Expected result:_
If error message appeared, search shouldn't start.

**Comment by Stref** [comment]
![747](https://cloud.githubusercontent.com/assets/13001938/8323087/63c7537e-1a48-11e5-809b-ccc0a44b25a0.png)

**Comment by ghost** [comment]
BCC - 26-Jun-2015 - Recommend: Fix

Issue fixed on my local build in changes made for issue #15

**Comment by ghost** [comment]
Review Meeting Findings - 01-Jul-2015

Already fixed for another issue #15

**Comment by Stref** [comment]
Validated on build from 14.08.2015
Issue fixed, search doesn't start when user noticed with warning message. Search starts only after user click Ok button (see screenshot attached)
![153](https://cloud.githubusercontent.com/assets/13001938/9270327/95aa9550-4279-11e5-84a5-a4563adea018.png)

Closed
